autowebcompat/utils.py

```diff
@@ -293,3 +293,34 @@ def write_train_info(information, model, train_history, file_name=None):
             row = train_history_list[i]
             for col in row:
                 print('%f\t\t' % col, end=' ', file=f)
+
+
+def create_screenshot_name(file_name_info):
+    if 'seq_no' in file_name_info.keys():
+        new_file_name = '_'.join([file_name_info['bug_id'], file_name_info['seq_no'], 'H', file_name_info['width'], 'V', file_name_info['height'], file_name_info['browser']])
+    else:
+        new_file_name = '_'.join([file_name_info['bug_id'], 'H', file_name_info['width'], 'V', file_name_info['height'], file_name_info['browser']])
+    return os.path.join(file_name_info['folder'], new_file_name + '.png')
+
+
+def parse_file_name(file_name):
+    file_name_parts = file_name.split('_')
+    file_info = {}
+    file_info['bug_id'] = int(file_name_parts[0])
+
+    shift = 0
+    if len(file_name_parts) >= 2 and file_name_parts[1].isdigit():
+        file_info['seq_no'] = int(file_name_parts[1])
+        shift = 1
+    if len(file_name_parts) > 2:
+        file_info['width'] = int(file_name_parts[shift + 2])
+        file_info['height'] = int(file_name_parts[shift + 4])
+    return file_info
+
+
+def create_dom_name(file_name_info):
```

Comment:
This function should reuse create_screenshot_name, since the format is the same except for the dom_ part

Reply:
But create_screenshot_name uses height and width also as parameters which is not for dom

Reply:
You can make them `None` by default and not add them when creating the name for dom

Reply:
Yeah! But it will result in some if else in create_file_name. I hope it will be fine in that case?

Reply:
Sure!